### ISSUE TITLE
how to apply  EL loss?

### ISSUE THREAD

**Issue by GinkgoX** [charrywhite/elastic_interaction_based_loss#2]
Dear charrywhite:
    Great work! could you give a biref usage about the EL loss?

**Comment by charrywhite** [comment]
> Dear charrywhite:
> Great work! could you give a biref usage about the EL loss?

Sorry for the late reply. You can use take  the softmax ouput of target class as the input of EL loss. I will upload code later.

**Comment by sjjdd** [comment]
But how to use it,if I use sigmoid.

**Comment by charrywhite** [comment]
> But how to use it,if I use sigmoid.

Hi,

I think sigmoid may not a good choice for this loss function. The fact that two channels of softmax output are interrelated guaranteed we can put only one channel map into elastic loss. While each channel of sigmoid is independent, I am not sure if it will work.

**Comment by charrywhite** [comment]
> But how to use it,if I use sigmoid.

Actually in binary classification sigmoid is equivalent to softmax, so there is no need to change to sigmoid.

**Comment by sjjdd** [comment]
> > But how to use it,if I use sigmoid.
> 
> Actually in binary classification sigmoid is equivalent to softmax, so there is no need to change to sigmoid.

OK，thanks very much
